### ISSUE TITLE
CQ dbrec creation is not steered to peer

### ISSUE THREAD

**Issue by drossetti** [gpudirect/libmlx5#5]

**Comment by drossetti** [comment]
@haggaie @ferasd would you review the prototype change ?

**Comment by haggaie** [comment]
This looks fine, but you will also need to change the kernel to forward mlx5_ib_db_map_user calls to the peer umem allocation and the peer client.

**Comment by drossetti** [comment]
@haggaie right, I have a patch for that on the pipeline, which will be pushed to a branch of mlnx-ofa_kernel

**Comment by drossetti** [comment]
@haggaie matching kernel part is https://github.com/gpudirect/mlnx-ofa_kernel/commit/efd43205b16c3ca716c8c806d3410a96f441b2fb
